tests/php/test-class-amp-comments-sanitizer.php

```diff
@@ -218,7 +218,7 @@ public function test_ampify_threaded_comments( $ampify_comment_threading, $comme
 			$this->assertInstanceOf( Element::class, $span );
 			$this->assertTrue( $span->hasAttribute( Amp::BIND_DATA_ATTR_PREFIX . 'text' ) );
 
-			$comment_reply_links = $dom->xpath->query( '//a[ @data-commentid and @data-postid and @data-replyto and @data-respondelement and contains( @class, "comment-reply-link" ) ]' );
+			$comment_reply_links = $dom->xpath->query( '//a[ @data-commentid and @data-postid and @data-respondelement and contains( @class, "comment-reply-link" ) ]' );
```

Comment:
`data-replyto` was added in WP 5.5 - https://github.com/WordPress/wordpress-develop/commit/251fd685dd932200087ad04fa49c598ef9657cda

Reply:
Because xpath query with `@data-replyto` will not work for WP < 5.5 because `data-reply` attribute was added in WP 5.5. Hence it will not get comment reply links on WP 5.3 and 5.4